### ISSUE TITLE
el[0].childNodes[Symbol.iterator] is not a function

### ISSUE THREAD

**Issue by chrisamow** [threema-ch/threema-web#64]
After connecting my samsung s4 (sgh-m919) android (4.4.4) to my chrome (Version 46.0.2490.71 (64-bit)) linux (mint 17) pc - it all shows, and I click a user to send a message to - and it populates my browser with recent messages.  I type a new message to send in the box at the bottom, but when I click the split triangle on the right (hover state even changes) - nothing happens - and the text remains in the box.

**Comment by chrisamow** [comment]
some interesting debug output from chrome dev:

```
Sending request/keyPersisted message
6angular.js:14199 TypeError: el[0].childNodes[Symbol.iterator] is not a function
    at app.js:9481
    at angular.js:19942
    at completeOutstandingRequest (angular.js:6045)
    at angular.js:6324(anonymous function) @ angular.js:14199
angular.js:14199 Sending message request for group 2 with message id null
angular.js:14199 Sending request/message message
angular.js:14199 Sending request/typing message
angular.js:14199 Sending message request for contact YWAD5HT9 with message id null
angular.js:14199 Sending request/message message
saltyrtc-task-webrtc.es5.js:349 [SaltyRTC.SecureDataChannel] Received chunk
saltyrtc-task-webrtc.es5.js:370 [SaltyRTC.SecureDataChannel] Decrypting incoming data...
angular.js:14199 New incoming message (9067 bytes)
angular.js:14199 Received message response
50angular.js:14199 TypeError: el[0].childNodes[Symbol.iterator] is not a function
    at app.js:9481
    at angular.js:19942
    at completeOutstandingRequest (angular.js:6045)
    at angular.js:6324(anonymous function) @ angular.js:14199
angular.js:14199 Sending read request for contact YWAD5HT9 (msg 3237)
angular.js:14199 Sending request/read message
angular.js:14199 Sending thumbnail request for contact 3232
angular.js:14199 Sending request/thumbnail message
3saltyrtc-task-webrtc.es5.js:349 [SaltyRTC.SecureDataChannel] Received chunk
saltyrtc-task-webrtc.es5.js:370 [SaltyRTC.SecureDataChannel] Decrypting incoming data...
angular.js:14199 New incoming message (148932 bytes)
angular.js:14199 Received thumbnail response
angular.js:14199 Sending request/typing message
angular.js:14199 TypeError: composeDiv[0].childNodes[Symbol.iterator] is not a function
    at submitText (app.js:8222)
    at sendText (app.js:8262)
    at app.js:8279
    at angular.js:19942
    at completeOutstandingRequest (angular.js:6045)
    at angular.js:6324(anonymous function) @ angular.js:14199
app.js:8252 Uncaught TypeError: composeDiv[0].childNodes[Symbol.iterator] is not a function
angular.js:14199 Sending request/typing message
app.js:8252 Uncaught TypeError: composeDiv[0].childNodes[Symbol.iterator] is not a function
2angular.js:14199 TypeError: composeDiv[0].childNodes[Symbol.iterator] is not a function
    at submitText (app.js:8222)
    at sendText (app.js:8262)
    at app.js:8279
    at angular.js:19942
    at completeOutstandingRequest (angular.js:6045)
    at angular.js:6324(anonymous function) @ angular.js:14199
28app.js:8252 Uncaught TypeError: composeDiv[0].childNodes[Symbol.iterator] is not a function
angular.js?v=1.0.1:14199 Sending read request for contact YWAD5HT9 (msg 3237)
angular.js?v=1.0.1:14199 Sending request/read message
```

**Comment by dbrgn** [comment]
Thanks, I've come across error reports like this more than once already.

At first, the error message looks like the browser is too old for ES2015 iterators. But since Chrome 46 should support that, we'll have to investigate.

**Comment by dbrgn** [comment]
@chrisamow does the issue still persist? If yes, could you try to open the debug toolbar (F12) and to paste the follwing code into the console window?

    Symbol.iterator

What does it return? On my machine, it looks like this:

![iterator](https://cloud.githubusercontent.com/assets/105168/23127544/b588dacc-f77b-11e6-837a-892434b8e378.png)

**Comment by dbrgn** [comment]
Apparently for-of loops on `NodeList` objects does not work on Chrome <51 and Firefox <50.

**Comment by chrisamow** [comment]
Sorry, missed that last update - I updated my main machine to ubuntu 16, I can try your iterator experiment on another old machine if it is still useful to you.  3maweb is working fine on chrome 56 as expected.

**Comment by dbrgn** [comment]
If you have a Chrome 50 instance to test that would be nice. I tried to download old builds, but couldn't get them to work on my machine. And compiling Chromium takes multiple hours :)

The fix is already in place at #98 though.
